Regression/Checksum/benchmarks_json/test_rsphere_langmuir_multi.json

```diff
@@ -1,28 +1,27 @@
 {
   "lev=0": {
-    "Bt": 4.955190029855237e-15,
-    "Er": 16181760914.279827,
-    "jr": 863095696601.4752,
-    "rho_electrons": 18595449.23275246,
-    "rho_ions": 18494177.078754093
+    "Bt": 4.521366416357738e-15,
```

Comment:
Would it be problematic (e.g., for post-processing tools) to update also this name from `Bt` to `Btheta` (as well as others, e.g., `Bp`, `Et`, `Ep`, etc.)?

Reply:
For plotfiles this would be no problem. Not sure about OpenPMD. Note that this could be a separate PR since the same change could be made to the theta component in RZ.